src/lib/components/injector.tsx

```diff
@@ -210,6 +212,25 @@ const elementsFabric = (type: PaymentElements): object => {
         };
     }
 
+    if (type === 'googlePay') {
+        /**
+         * Apple Pay
+         */
+
+        const GooglePayElement = Hoc(
+            'GooglePayElement',
+            GooglePayElementComponent,
+            (data: FramePayContext) =>
+                ({
+                    Rebilly: makeRebillyProps(data)
+                } as GooglePayProps)
+        );
+
+        return {
+            GooglePayElement
+        };
+    }
+
     if (type === 'digitalWallet') {
```

Comment:
Shouldn't the new google pay stuff replace all the digital wallet files, like in the framepay-vue PR?

Reply:
Sorry for my rush, I removed all the `digital wallet` concepts.